qemu/tests/luks_image_over_qsd.py

```diff
@@ -0,0 +1,107 @@
+from provider.qsd import QsdDaemonDev
+
+from virttest import error_context
+from virttest import qemu_storage
+from virttest import utils_numeric
+from virttest import data_dir
+from virttest import utils_disk
+from virttest import storage
+from virttest.qemu_monitor import QMPCmdError
+
+
+# This decorator makes the test function aware of context strings
+@error_context.context_aware
+def run(test, params, env):
+    """
+        Export a luks image via QSD with NBD inet.
+        1. Create a luks image.
+        2. Start a QSD daemon, and export the luks image with NBD inet.
+        3. Check the info of the image over NBD.
+        4. Boot up a guest with the exported image as a data disk.
+        5. Check there is the data disk in the guest.
+        6. Connect to the QMP sock
+        6.1 Query the block exports
+        6.2 luks key management.
```

Comment:
Suggest you add "stop the QSD daemon".  Suggest also update it in the commit message.